src/metatensor/models/cli/train_model.py

```diff
@@ -201,6 +200,7 @@ def train_model(options: DictConfig) -> None:
         outputs=outputs,
     )
 
+    logger.info("Calling model trainer")
```

Comment:
What is our syntax. When do we call it model and when architecture? I thought model is really the trained or to be trained pytorch object and everything else we call architecture.

```suggestion
    logger.info("Calling architecture trainer")
```

Reply:
Ok!